packages/plugins/test/PostDeploymentActions.spec.ts

```diff
@@ -643,7 +643,7 @@ describe('Post-Deployment Actions', () => {
     })
   })
 
-  const integrations: Array<Integration> = ['hardhat', 'foundry']
+  const integrations: Array<Integration> = ['hardhat']
```

Comment:
Is there a reason you removed 'foundry' here? Without this, the test suite won't be run against the foundry deploy task